app/src/main/java/cm/aptoide/pt/ads/InterstitialClick.java

```diff
@@ -0,0 +1,5 @@
+package cm.aptoide.pt.ads;
+
+public enum InterstitialClick {
+  INTERSTITIAL_LOADED, INTERSTITIAL_CLICKED, INTERSTITIAL_FAILED, INTERSTITIAL_SHOWN, INTERSTITIAL_DISMISSED;
```

Comment:
INTERSTITIAL_FAILED, INTERSTITIAL_SHOWN, INTERSTITIAL_DISMISSED 
are not being used.

Reply:
fixed